rita_client/src/dashboard/exits.rs

```diff
@@ -65,20 +65,22 @@ pub fn dashboard_get_exit_info() -> Result<Vec<ExitInfo>, RitaClientError> {
 
                     for exit in exit_client.exits.clone().into_iter() {
                         let selected = is_selected(&exit.1, current_exit);
-                        let have_route = if let Some(selected_ip) = get_current_selected_exit() {
-                            do_we_have_route(&selected_ip, &route_table_sample)?
+                        let current_selected_ip = get_current_selected_exit();
+                        let route_ip = if let Some(c_ip) = current_selected_ip {
+                            if exit.1.subnet.contains(current_selected_ip.unwrap()) {
```

Comment:
can we add some sort of utility function for getting a selected exit it seems we're using it pretty often.